scripts/detect_errors.py

```diff
@@ -1,9 +1,30 @@
 import itertools
+from json import loads
+from pathlib import Path
 import sys
 
+def format_msg(msg):
+    return f"{msg['file_name']}:{msg.get('pos_line')}:{msg.get('pos_col')}: {msg.get('severity')}: {msg.get('text')}\n"
+
+def write_and_print_noisy_files(noisy_files):
+    with open('src/noisy_files', 'w') as f:
+        for file in noisy_files:
+            f.write(file + '\n')
+            print(file)
+
+noisy_files = set()
 for line in sys.stdin:
-    sys.stdout.write(line)
-    if 'error' in line:
-        for line in itertools.islice(sys.stdin, 20):
-            sys.stdout.write(line)
+    msg = loads(line)
+    sys.stdout.write(format_msg(msg))
```

Comment:
```suggestion
    print(format_msg(msg))
```
Or otherwise `sys.stdout.flush()`.